File: app/src/main/java/de/fhkiel/temi/robogguide/database/Media.java
package de.fhkiel.temi.robogguide.database;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable(tableName = "media")
public class Media {

    @DatabaseField(generatedId = true)
    private int id;

    @DatabaseField
    private String url;

    @DatabaseField
    private int texts_id;

    public Media() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public int getTexts_id() {
        return texts_id;
    }

    public void setTexts_id(int texts_id) {
        this.texts_id = texts_id;
    }
}

File: app/src/main/java/de/fhkiel/temi/robogguide/database/Media.kt
package de.fhkiel.temi.robogguide.database

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

/**
 * Orm Data class to get the Media table
 *
 * @property id
 * @property url
 * @property texts_id
 */
@DatabaseTable(tableName = "media")
data class Media (
    @DatabaseField(generatedId = true, columnName = "id")
    val id: Int,

    @DatabaseField(canBeNull = false, columnName = "url")
    val url: String,

    @DatabaseField(canBeNull = false,  columnName = "texts_id")
    val texts_id: Int
)
{  constructor() : this(0,"",0) }

